src/templates/_includes/forms/date.twig

```diff
@@ -5,6 +5,7 @@
 {% set outputTzParam = outputTzParam ?? true %}
 {% set isMobile = craft.app.request.isMobileBrowser %}
 {% set isDateTime = isDateTime ?? false %}
+{% set labelledBy = fieldset ?? null %}
```

Comment:
Not following this – `fields` is going to be `true` or `false`, and `labelledBy` is supposed to be set to a DOM element ID. (Ditto for `time.twig`.)

Reply:
@brandonkelly I think I meant `{% set labelledBy = fieldset ? null : labelledBy %}`; on second thought, that’s not exactly what I wanted to accomplish.

The issue I’m having is that just for the metafields, I want the `aria-labelledby` value to go away since it overwrites the explicit Date and Time labels being set by `aria-label`.

<img width="824" alt="Screenshot 2023-01-19 at 12 02 35 PM" src="https://user-images.githubusercontent.com/17525979/213547761-f8468a5d-8283-4609-8ed1-ffce7a72e82b.png">

I was looking at other instances of Datetime, Date, and Time fields in the field layout, and `aria-labelledby` isn’t being set anywhere else, so I wasn’t sure the best place to make this change to have it be specific to just the Post Date and Post Time.